### DISCUSSION TITLE
Is some setup still needed when using the GitHub app?

### DISCUSSION THREAD

**Discussion by flexagoon** [WeblateOrg/weblate#11565] [Q&A]
I'm setting up Weblate with my GitHub project.

The "Automatically receiving changes from GitHub" section has this sentence:

> If you are using Hosted Weblate, the recommended approach is to install the [Weblate app](https://github.com/apps/weblate), that way you will get the correct setup without having to set much up. It can also be used for pushing changes back.

If I'm using that app, do I still need to set up webhooks and stuff? Or is just installing it enough to get Weblate to work?

**Comment by nijel** [comment]
Webhooks are done via app, but user still needs to be added for pushing changes from Weblate (until https://github.com/WeblateOrg/weblate/issues/1688 is addressed).

**Reply by rfc2822** [reply]
Hello,

What does it mean that the _user needs to be added for pushing changes_? Which user needs to be added where? The documentation doesn't say anything about that.

We're using Hosted Weblate. It seems to detect then the repository is out of date and how many commits it's behind, but it doesn't merge the new source commits automatically.

So I also have exactly the same question:

> If I'm using that app, do I still need to set up webhooks and stuff? Or is just installing it enough to get Weblate to work?

Any help appreciated.

**Reply by nijel** [reply]
https://github.com/WeblateOrg/weblate/pull/18910 should clarify this in our docs.
